### PR TITLE
Add default automations to core feed

### PR DESCRIPTION
**Pull request by alexk307**
# Description

https://www.notion.so/sublimesecurity/Add-Core-Automations-to-the-Feed-13404655fc9d807faff3e92303fcf8ee?pvs=4

https://sublimesecurity.slack.com/archives/C07D63BFSEL/p1732546652799719?thread_ts=1732546334.120099&cid=C07D63BFSEL

### CODE REVIEW

File: automations/remediate_messages_via_herd_immunity.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: "Remediate messages via herd immunity"
```
Comment (ianthiel) [inline automations/remediate_messages_via_herd_immunity.yml:1]: per discussion [here](https://sublimesecurity.slack.com/archives/C07D63BFSEL/p1731943731270829?thread_ts=1731528259.775899&cid=C07D63BFSEL), this should be:

name: "Remediate user reports via herd immunity"

Comment (alexk307) [reply automations/remediate_messages_via_herd_immunity.yml:1]: thanks I'll update the Notion task too.

File: .github/workflows/rule-validate.yml [hunk 2]
```diff
@@ -123,6 +123,15 @@ jobs:
...
 
+#          for f in automations/*.yml
+#          do
+#            counter=$((counter + 1))
+#            yq -o=json eval "$f" >> bulk_validate_request.json
+#            if [[ $counter -ne $file_count ]]; then
+#              echo "," >> bulk_validate_request.json
+#            fi
+#          done
+
```
Comment (jkamdjou) [inline .github/workflows/rule-validate.yml:134]:
```suggestion
#          for f in automations/*.yml
#          do
#            counter=$((counter + 1))
#            yq -o=json eval "$f" >> bulk_validate_request.json
#            if [[ $counter -ne $file_count ]]; then
#              echo "," >> bulk_validate_request.json
#            fi
#          done

```
is this meant to be here? this is commented out right?

File: automations/remediate_malicious_user_reports.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+name: "Remediate malicious user reports"
+description: "Remediate user reported messages with a Malicious Attack Score."
```
Comment (jkamdjou) [inline automations/remediate_malicious_user_reports.yml:2]:
```suggestion
description: "Quarantine user reported messages with a Malicious Attack Score."
```

File: automations/remediate_malicious_messages.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+name: "Remediate malicious flagged messages"
+description: "Remediate flagged messages with a Malicious Attack Score."
```
Comment (jkamdjou) [inline automations/remediate_malicious_messages.yml:2]:
```suggestion
description: "Quarantine flagged messages with a Malicious Attack Score."
```

File: .github/workflows/rule-validate.yml (hunk 2 above)
Comment (alexk307) [reply .github/workflows/rule-validate.yml:134]: yes - it was from the `automations-expterimental` branch that we are using to test with. There's a slight issue with validation (triage vs. triage_rule) that I need to fix in the future. I was going to keep this here to remind me.

Comment (alexk307) [reply .github/workflows/rule-validate.yml]: I rm'd it for now. I'll FUP with a task.

**Review by jkamdjou — APPROVED** [review]

**Pull request merged**